Misc/NEWS.d/next/Library/2025-09-06-08-29-08.gh-issue-138568.iZlalC.rst

```diff
@@ -0,0 +1,2 @@
+In :mod:`pydoc` interactive help mode to handle empty input correct as doc.
+press Enter without input should continue in help session instead of exiting.
```

Comment:
I think this wording is still difficult to understand.  Here's one possible rephrasing:

```suggestion
Adjusted the built-in :func:`help` function so that empty inputs are ignored in
interactive mode.
```

Reply:
Nice hint thanks